features/homepage/viewing_only_enabled_taxons_in_taxon_menu.feature

```diff
@@ -0,0 +1,29 @@
+@homepage
+Feature: Viewing only enabled taxons in taxon menu
+    In order to not access disabled taxons
```

Comment:
Ok, good point. I am ok in changing this only in the that specific line of the feature definition. Because i have some doubts in rewording this in the other parts of this feature: the taxon availability is not only related to the fact that it is enabled but even to the fact that it should be associated to the current channel. This features only test enabled/disabled taxon and not their belonging to a specific channel.
I don't know if i expressed correctly what i am thinking =)